dist/src/main/java/io/camunda/zeebe/broker/BrokerModuleConfiguration.java

```diff
@@ -42,13 +42,15 @@
 @Configuration(proxyBeanMethods = false)
 @ComponentScan(
     basePackages = {
+      "io.camunda.commons",
       "io.camunda.zeebe.broker",
       "io.camunda.zeebe.shared",
       "io.camunda.zeebe.gateway.rest",
       "io.camunda.authentication"
     })
 @ConfigurationPropertiesScan(
     basePackages = {
+      "io.camunda.commons",
```

Comment:
🤔 didn't you removed this in the other pr?

Reply:
Yep, the PRs are built upon each other. This intermediate step introduces this base package and removes it later. I tried to keep the PRs focused to better catch any issues.

Reply:
I see 👍🏼